db.json

```diff
@@ -5274,7 +5419,7 @@
   },
   "application/vnd.smaf": {
     "source": "iana",
-    "extensions": ["mmf"]
+    "extensions": ["mmf","smaf"]
```

Comment:
.smaf not on https://www.iana.org/assignments/media-types/application/vnd.smaf

Add in https://gitlab.freedesktop.org/xdg/shared-mime-info/-/commit/343b7b9a9304c322b3fc7dc60057de233f6e6662

None at https://github.com/search?q=path%3A*.smaf&type=code

I dont see a lot of evidence of .smaf being used